Core/clim-core/gadgets.lisp

```diff
@@ -2866,14 +2866,23 @@ if INVOKE-CALLBACK is given."))
     ((or hbox-pane hrack-pane) :vertical)
     ((or vbox-pane vrack-pane) :horizontal)))
 
+(cl:defconstant +box-adjuster-gadget-major-size+ 6)
+(cl:defconstant +box-adjuster-gadget-minor-size+ 1)
```

Comment:
this is just a note (not change request) - we should find where we depend on non-conforming defconstant, fix these places and do not patch defconstant in climi.